test/test_av.py

```diff
@@ -111,9 +112,9 @@ def norm_indicator(expected, dcoll, soln, **kwargs):
 
     # get meshmode polynomials
     group = dcoll.discr_from_dd("vol").groups[0]
-    basis = group.basis()  # only one group
+    basis = group.basis_obj().functions  # only one group
     unit_nodes = group.unit_nodes
-    modes = group.mode_ids()
+    modes = group.basis_obj().mode_ids
```

Comment:
Not sure if it matters here, but this `basis_obj()` change appears to fail for quadrature discretizations.

Reply:
Hmm... Can you explain more? Not sure if I follow what's failing.